pipeline.yaml

```diff
@@ -671,7 +671,7 @@ deploymentSpec:
         command:
         - /bin/sh
         - -c
-        image: registry.redhat.io/ubi9/toolbox@sha256:da31dee8904a535d12689346e65e5b00d11a6179abf1fa69b548dbd755fa2770
+        image: registry.access.redhat.com/ubi9/toolbox
```

Comment:
all images need to match: https://github.com/red-hat-data-services/ilab-on-ocp/blob/rhoai-2.17/utils/consts.py#L2

it looks like it correctly matched it before, why change it?